runtime/shiden/src/lib.rs

```diff
@@ -1127,6 +1140,38 @@ impl OnRuntimeUpgrade for RecalculationEraFix {
     }
 }
 
+pub struct SetNewTierConfig;
+impl OnRuntimeUpgrade for SetNewTierConfig {
+    fn on_runtime_upgrade() -> Weight {
+        use astar_primitives::oracle::PriceProvider;
+        use frame_support::BoundedVec;
+
+        // Set new init tier config values according to the forum post
+        let mut init_tier_config = pallet_dapp_staking_v3::TierConfig::<Runtime>::get();
+        init_tier_config.number_of_slots = 55;
+        init_tier_config.slots_per_tier =
+            BoundedVec::try_from(vec![2, 11, 16, 24]).unwrap_or_default();
+
+        #[cfg(feature = "try-runtime")]
+        {
+            assert!(
+                init_tier_config.number_of_slots >= init_tier_config.slots_per_tier.iter().sum::<u16>() as u16,
+                "Safety check, sum of slots per tier must be equal or less than max number of slots (due to possible rounding)"
```

Comment:
Why this can be `>` ? Should we enforce `=` ?

Reply:
Initially I've put it like that but forgot about the rounding, or flooring to be precise.

We might lose a slot per tier due to this.

Reply:
<img width="1049" alt="Screenshot 2024-02-23 at 15 36 37" src="https://github.com/AstarNetwork/Astar/assets/3002868/132d9dc2-40b2-4acc-b1fb-c1b9feb1ef7a">